app/assets/images/delete-icon.svg

```diff
@@ -0,0 +1 @@
+<svg id="Capa_1" enable-background="new 0 0 515.556 515.556" height="512" viewBox="0 0 515.556 515.556" width="512" xmlns="http://www.w3.org/2000/svg"><path d="m64.444 451.111c0 35.526 28.902 64.444 64.444 64.444h257.778c35.542 0 64.444-28.918 64.444-64.444v-322.222h-386.666z"/><path d="m322.222 32.222v-32.222h-128.889v32.222h-161.111v64.444h451.111v-64.444z"/></svg>
```

Comment:
Where did this icon come from?

Reply:
I downloaded this icon from w3c website to display delete icon

Reply:
Where exactly? If it's public domain or freely licensed, we need to add credits for the author and source in miscellaneous_credits.md; if it's not, we can't use it and should find an alternative.

Reply:
To be precise I used icon from here [https://www.flaticon.com/free-icon/delete_2089743?term=delete&page=1&position=11](https://www.flaticon.com/free-icon/delete_2089743?term=delete&page=1&position=11)

They have mentioned there its free for personal and commercial use with attribution